docs/data_management/data_responsibilities.md

```diff
@@ -0,0 +1,82 @@
+
+{{ read_csv('data_management/res/cheaha_project_directory.csv', keep_default_na=False) }}
+
+## Individual Allocation
+
+An individual allocations is intended for personal or individual use and is available to all UAB affiliated individuals or UAB employee's sponsored Collaborator. It is tied to the individual’s email and provide 5 TB of home/user directory on Cheaha and additional 5 TB of LTS allocation.
+
+{{ read_csv('data_management/res/cheaha_individual_account.csv', keep_default_na=False) }}
+
+## Data Archival and Backup Procedures
+
+Proper data archival and backup practices ensure efficient storage utilization and data protection. IT is the responsibility of researchers and users of Cheaha to organize data, archive inactive files, and back up critical data.
+
+### Archival
+
+<!-- markdownlint-disable MD046 -->
+!!! important
+
+    Archival of data is the responsibility of researchers using Cheaha.
+<!-- markdownlint-enable MD046 -->
+
+At this time, Research Computing does not offer a method of archival. If you have need for archival, please feel free to contact [Support](../help/support.md) to start a conversation.
+
+A possible external resource for archival is available through University of Oklahoma (OU) Supercomputing Center for Education and Research (OSCER). Please see the following link for details: <https://www.ou.edu/oscer/resources/ourrstore--ou---regional-research-store>.
+
+### Backups
+
+<!-- markdownlint-disable MD046 -->
+!!! important
+
+    Backups of data are the responsibility of researchers using Cheaha.
+<!-- markdownlint-enable MD046 -->
+
+A good practice for backing up data is to use the 3-2-1 rule, as [recommended by US-CERT](https://www.cisa.gov/sites/default/files/publications/data_backup_options.pdf):
+
+- **3**: Keep **3** copies of important data. 1 primary copy for use, 2 backup copies.
+- **2**: Store backup copies on **2** different media types to protect from media-specific hazards.
+- **1**: Store **1** backup copy offsite, located geographically distant from the primary copy.
+
+What hazards can cause data loss?
+
+- Accidental file deletion.
+    - Example: mistakenly deleting the wrong files when using the [shell command](../workflow_solutions/shell.md#delete-files-and-directories-rm-rmdir) `rm`.
+    - Files deleted with `rm` or any similar command can not be recovered by us under any circumstances.
+    - Please restore from a backup.
+- Natural disasters.
+    - Examples: tornado; hurricane.
+    - All of our data sits in one geographical location at the UAB Technology Innovation Center (TIC).
+    - Plans to add geographical data redundancy are being considered.
+    - Please restore from an offsite backup.
+- Unusable backups.
+    - Examples: backup software bug; media destroyed; natural disaster at offsite location.
+    - Regularly test data restoration from all backups.
+
+How can I ensure data integrity?
```

Comment:
Is this about data integrity, or about backup options?

Reply:
I think we may need to remove this. The fact of the matter is we don't have any good backup options to offer right now. I would simply say they need to contact us if they have backup needs and we can discuss options with them, depending on their use case. We want to engage in a conversation.